server/src/main/resources/application.properties

```diff
@@ -10,18 +10,20 @@ springdoc.api-docs.enabled=false
 kitu.kotoutumiskoulutus.koealusta.wstoken=${KIELITESTI_TOKEN}
 kitu.kotoutumiskoulutus.koealusta.scheduling.enabled=false
 
-kitu.oppijanumero.username=
 kitu.oppijanumero.password=${OPPIJANUMERO_PASSWORD}
-kitu.oppijanumero.callerid=
-kitu.oppijanumero.casUrl=
-kitu.oppijanumero.serviceUrl=
+kitu.oppijanumero.username=koto-rekisteri
+kitu.oppijanumero.callerid=1.2.246.562.24.85478397072
+kitu.oppijanumero.casUrl=https://virkailija.untuvaopintopolku.fi/cas
+kitu.oppijanumero.serviceUrl=https://virkailija.untuvaopintopolku.fi/oppijanumerorekisteri-service
 
 db-scheduler.enabled=true
 db-scheduler.delay-startup-until-context-ready=true
 
 kitu.yki.scheduling.enabled=false
 kitu.yki.scheduling.import.schedule=-
-kitu.yki.baseUrl=
+kitu.yki.baseUrl=https://yki-test.cc.jyu.fi/yki-sp/oph/
```

Comment:
Pitäisikö tämä konfiguroida toisin päin, eli `baseUrl` tyhjäksi tässä ja asettaa ei-tyhjään arvoon ympäristökohtaisesti? YKI testiympäristön käyttäminen oletusarvona tuntuu hieman hassulta.